akmodsposttrans@.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Building and installing kmods for kernel %i
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/usr/bin/systemd-inhibit --who=akmods --why="Building kernel modules" /usr/sbin/akmods --from-kernel-posttrans --kernels %i
```

Comment:
I believe so, I originally added Conflict=shutdown.target not knowing that all services get this by default. I believe it only stops units from starting once a shutdown has been initiated or asks it to shut down. 

The more I think about it the more I think it should be changed to type oneshot since we don't provide an ExecStop.